### DISCUSSION TITLE
Symbol definitions - example block clearing point definition

### DISCUSSION THREAD

**Discussion by sixdiamants** [railtoolkit/tikz-trackschematic#11] [Q&A]
is there a public definition of symbols ? 

E.g. what is a  block clearing point? 
In EULYNX DataPrep we define [the block element](https://dataprep.eulynx.eu/2022-01/EARoot/EA2/EA4/EA1/EA1/EA4826.htm) and the block clearance trigger. I suspect that the clearance point is the latter.

**Comment by kaat0** [comment]
Currently not. But I suppose there should be one:

The CLEARING POINT (CP) is "A point a train must have cleared completely before a signal or a route may be released." (source: http://joernpachl.de/glossary.htm#CLEARING_POINT)
Ergo, the BLOCK CLEARING POINT (BCP) is a point a train must have cleared completely before a _block_ signal may be released.
And, the ROUTE CLEARING POINT (RCP) is a point a train must have cleared completely before a _route_ signal may be released.
BCP and RCP can of course also be located on the same spot.

The differentiation between block clearing point and route clearing point probably comes from the Dresden Railway School (Faculty "Friedrich List"). There it was a common differentiation with SZS (german: Signalzugschlussstelle) and FZS (german: Fahrstraßenzugschlussstelle).
So far, I have only found it indirectly in the English-language literature: the UIC Codex 406 (2004) shows the BCP/SZS on page 13 in figure 5 as "contact". But because the Codex 406 does not know any routes across turnouts, there is no RCP/FZS there.
In the book "Railway Operation and Control" by Jörn Pachl (2018), the position of the BCP at the end of the "S" can be found on page 52 in Figure 3.1 c). Where "S" is the protective section behind the block signal, which must be kept clear for safety reasons and is also known as the overlap (compare Figure 3.5 page 60). Once turnouts are included, there may be further clearing points to ensure that the turnouts are clear (see Figure 4.7 page 130). Figure 4.45 on page 175 shows the interaction between BCP and RCP. On the left side of the station, the BCP comes after signal 13 and the RCP at the end of the two turnouts. For signal 15, the BCP and the RCP are on the same spot.

Some of the Figures can also be found in the book "Railway Timetabling & Operations" by I.A. Hansen & J. Pachl (Eds., 2014)
Figure 3.5 in "Railway Operation and Control" is Figure 2.7 in "Railway Timetabling & Operations" page 22
Figure 4.45  in "Railway Operation and Control" is Figure 2.15 in "Railway Timetabling & Operations" page 31

**Comment by kaat0** [comment]
We are working on a public glossary for the definitions, symbols, and terms.
